tasks/configure.js

```diff
@@ -91,6 +92,10 @@ function task (opts) {
 		})
 		.then(function () {
 			console.log(target + " config vars are set");
+
+			return shellpromise('heroku labs:enable runtime-dyno-metadata --app ' + target, { verbose: true })
+				.then(() => console.log('Heroku metadata enabled'))
+				.catch((e) => console.error('Heroku metadata could not be enabled:', e.toString()));
```

Comment:
It shouldn't - the error is caught and will return `undefined`, same as `Promise.resolve()`